docs/tutorial/tutorial.md

```diff
@@ -0,0 +1,126 @@
+# Tutorial
+
```

Comment:
I'm missing a "what you'll learn" section here

Reply:
Added a "what you'll learn" section with reference to tutorial steps!